### PR TITLE
Use rossetmaster from jsk_tools

### PR DESCRIPTION
**Pull request by 130s**
Taking over https://github.com/tork-a/baxter_seminar/pull/17 

(mainly because I forked the repository on my own github organizaiton while the PR was still active. Sorry for non-essential disturbance).

### CODE REVIEW

File: baxter_seminar_helper/seminar.sh [hunk 1]
```diff
@@ -2,25 +2,10 @@
...
-       export PS1="\[\033[00;31m\][$ROS_MASTER_URI][$ROS_IP]\[\033[00m\] ${PS1}"
-       echo -e "\e[1;31mset ROS_MASTER_URI to $ROS_MASTER_URI\e[m"
-   }
-fi
-
 for (( i=1; i<=3; i++)) 
 do
   echo "#${i} ntpdate call";
-  ntpdate -q 011402P0006.local;
+  rossetmaster 011402P0006.local;
```
Comment (k-okada) [inline baxter_seminar_helper/seminar.sh]: if you use rossetmaster, you do not have to run this for 3 times

Comment (130s) [inline baxter_seminar_helper/seminar.sh]: Commit updated. It was wrong to replace `ntpdate` with `rossetmaster`.

**Comment by k-okada** [comment]
I'm afraid that this script does not work:
1)`for do;` must be end with `done`, this must be syntax error https://github.com/tork-a/baxter_seminar/blob/master/baxter_seminar_helper/seminar.sh#L12
2) if you write setup.sh into your bashrc, every time you start terminal, it runs ntpdate for 3 times, and every ntpdate usually takes few second, is that what you expected?
3) at least, at my computer, you have to have root permission to run ntpdate, so you have to run ntpdate with sudo,

my proposal is
- set rospi and rosmater at bashrc is ok.
- running ntpdate within bashrc is not a good idea
- runnig ntpdate manually, if you think 01... hostaname is too complex, set dns and deliver alias name such as baxter, or if you only have less than 5 student, you can write /etc/hosts for all students, that's wont take a few minutes.

**Comment by 130s** [comment]
> I'm afraid that this script does not work:

You are right. At least rossetmaster works though.
```
Set ROS master: 011402P0006.local
set ROS_MASTER_URI to http://011402P0006.local:11311
bash: /home/baxter/ros_ws/src/baxter_seminar/baxter_seminar_helper/seminar.sh: 行 17: 構文エラー: 予期しないファイル終了 (EOF) です
[http://011402P0006.local:11311][] baxter@u12-01:~$ 
```

**Comment by 130s** [comment]
https://github.com/jsk-ros-pkg/jsk_common/pull/845

**Pull request merged**